src/test/unit_test/multi_runner.cpp

```diff
@@ -348,6 +364,30 @@ multi_runner_base<IsParent>::message_queue_send(
     message_queue_->send(s.c_str(), s.size(), /*priority*/ 0);
 }
 
+template <bool IsParent>
+std::size_t
+multi_runner_base<IsParent>::tests() const
+{
+    return inner_->tests();
+}
+
+template <bool IsParent>
+std::size_t
+multi_runner_base<IsParent>::suites() const
+{
+    return inner_->suites();
+}
+
+template <bool IsParent>
+void
+multi_runner_base<IsParent>::add_failures(std::size_t failures)
```

Comment:
Can/Should we make `failures` pass by const ref?

Reply:
The general rule of thumb that the team uses is that if the item is the same size or smaller than a pointer, especially if it's a simple type, just pass by value. So we can, but we generally don't.